docs/resource/equinix_metal_project_ssh_key.examples.md

```diff
@@ -114,10 +114,10 @@ package generated_program;
 import com.pulumi.Context;
 import com.pulumi.Pulumi;
 import com.pulumi.core.Output;
-import com.pulumi.equinix.metal.ProjectSshKey;
-import com.pulumi.equinix.metal.ProjectSshKeyArgs;
-import com.pulumi.equinix.metal.Device;
-import com.pulumi.equinix.metal.DeviceArgs;
+import com.equinix.equinix.metal.ProjectSshKey;
```

Comment:
The old docs were wrong, but so are the new ones.  To keep this PR small, I'm tempted to use `sed` to inject the old, wrong package to avoid this diff for now and then update `sed` to inject the new, correct package in a later PR.